src/main.ts

```diff
@@ -244,6 +245,11 @@ const registerEventHandlers = () => {
     });
 
     ipcMain.handle('get-app-version', () => app.getVersion());
+
+    ipcMain.on('start-sleep-blocker', () => {
+        const id = powerSaveBlocker.start('prevent-app-suspension');
+        ipcMain.on('stop-sleep-blocker', () => powerSaveBlocker.stop(id));
+    });
```

Comment:
This creates a memory leak since the `stop-sleep-blocker` listeners never get removed.

Also, calling(?) `stop-sleep-blocker` when the blocker isn't currently active should be a noop.

```suggestion
    let blockerId: number | undefined;
    ipcMain.on('start-sleep-blocker', () => {
        if (blockerId === undefined) {
            blockerId = powerSaveBlocker.start('prevent-app-suspension');
        }
    });
    ipcMain.on('stop-sleep-blocker', () => {
        if (blockerId !== undefined) {
           powerSaveBlocker.stop(blockerId);
           blockerId = undefined;
        }
    });
```